src/mscorlib/shared/System/Type.cs

```diff
@@ -349,7 +349,7 @@ public virtual Array GetEnumValues()
         public static Type MakeGenericMethodParameter(int position)
         {
             if (position < 0)
-                throw new ArgumentException(SR.ArgumentOutOfRange_MustBeNonNegNum, nameof(position));
+                throw new ArgumentException(SR.Format(SR.ArgumentOutOfRange_MustBeNonNegNum, nameof(position)), nameof(position));
```

Comment:
Use `ArgumentOutOfRange_NeedNonNegNum` that we are using on majority of places instead?